sdk/communication/communication-phone-numbers/test/snippets.spec.ts

```diff
@@ -3,11 +3,13 @@
 
 import { AzureKeyCredential } from "@azure/core-auth";
 import {
+  AvailablePhoneNumber,
+  BrowseAvailableNumbersRequest,
   PhoneNumberCapabilitiesRequest,
   PhoneNumbersClient,
   SearchAvailablePhoneNumbersRequest,
   SipRoutingClient,
-} from "../src/index.js";
+} from "@azure/communication-phone-numbers";
```

Comment:
not blocking but I'd prefer changing back to `../src/index.js` later to have immediate feedback from source code changes without having to rebuild.  Previous update-snippets issue on Windows should be fixed now.